### DISCUSSION TITLE
Toggle centered-column layout on and off (centered-2-columns??)

### DISCUSSION THREAD

**Discussion by druskus20** [YaLTeR/niri#839] [Ideas]
My workflow mostly involves working with full-screen (or almost fullscreen) windows by default. 
I use `center-focused-column "always"` but I have problems whenever I want to see two windows side to side. 

I propose a way to toggle "center-focused-column" on and off - or maybe even a way to focus 2 columns at a time (side by side view, where the columns align to the center instead of the left border of the screen).

**Comment by YaLTeR** [comment]
You could toggle it by un-/commenting the line in the config programmatically.

But yeah, this is one of the reasons why I don't use always-center: frequently work side by side. If I need something centered temporarily, I press center-focused-column.

Not sure what good solution there is without overcomplicating it.

**Reply by druskus20** [reply]
Actually - what about a `center-focused-column "on-single-window"` option? It's not exactly the same but it would mostly achieve what I would like. 

- center 1 window workspaces 
- do not center when there's more than one window, which allows for side-by-side view

**Reply by YaLTeR** [reply]
This? https://github.com/YaLTeR/niri/wiki/Configuration:-Layout#always-center-single-column

**Comment by druskus20** [comment]
Oh, yes!
I think that will work
